src/bin/nydus-image/core/chunk_dict.rs

```diff
@@ -91,7 +92,12 @@ impl ChunkDict for HashChunkDict {
 }
 
 impl HashChunkDict {
-    fn from_bootstrap_file(path: &Path) -> Result<Self> {
+    fn from_bootstrap_file(
+        path: &Path,
+        target_rafs_version: Option<RafsVersion>,
+        target_compressor: Option<compress::Algorithm>,
+        target_digester: Option<digest::Algorithm>,
```

Comment:
It may be better to group these three fields into a struct for simplify.

Reply:
OK, thanks!